### DISCUSSION TITLE
Powermeter on Oled display

### DISCUSSION THREAD

**Discussion by darksideofthemoon1712** [hoylabs/OpenDTU-OnBattery#620] [Ideas]
is it possible to see the powermeter on the display?

**Comment by spcqike** [comment]
Right now it’s not possible. However you can create a fork and compile it yourself. That’s what I did. 

https://github.com/spcqike/OpenDTU-OnBattery/blob/oled_changes/src/Display_Graphic.cpp#L160

 my fork isn’t up to date so you can’t use it, if you want or need all the newer changes. But I think you get an idea.

**Comment by thinklerde** [comment]
Unfortunately this doesn't work for me. Try to integrate this into Visual Studio. But there is an error when compiling.
Unfortunately, I don't have the knowledge to do this on my own and I've been trying for several hours. Is there a way to get a bin file for dummies?

**Reply by darksideofthemoon1712** [reply]
Hallo thinklerde schein leider für dir anderen nicht so interresant. Ich würde mir das auch wünschen alle Daten auf dem Display zu sehen.
Leider ist die Zeit meines Programmierens schon einige Jahre her. Mit Visual Studio habe ich noch nie gearbeitet, damals hat man das noch mit Atom compiliert. Ich hoffe das es ingendwan doch kommt.

**Reply by thinklerde** [reply]
Scheint leider wirklich so. Ich bekomme es leider nicht hin, dazu hab ich vom Programmieren zu wenig Ahnung und momentan zu wenig Zeit mich da weiter rein zu arbeiten. Vllt. findet sich ja doch noch Jemand, der das umsetzen mag. Ich kann mir eigentlich nicht vorstellen, dass nur wir zwei diesen Wunsch haben. Oder doch?

**Comment by schlimmchen** [comment]
@darksideofthemoon1712 @spcqike @thinklerde Your feedback for #644 is welcome.
